charts/csi-secrets-store-provider-azure/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{ if .Values.rbac.install }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-secrets-store-provider-azure
```

Comment:
Where do we use this serviceAccount in the chart deployment yamls?

Reply:
Had added it in `deployment/*` dir, missed out the charts. Updated!